CONTRIBUTING.md

```diff
@@ -0,0 +1,56 @@
+# Contributing to Foreman Documentation
+
+The Foreman Community welcomes all feedback, issues, pull requests (PRs), and reviews.
+Every contributor has different backgrounds, interests, and experiences with Foreman and its open source community.
+
+We give ourselves these guidelines to collaborate and contribute to Foreman documentation more efficiently.
+It helps everyone to set expectations, communicate conventions, and ensures a happy collaborative work across organisations and backgrounds.
+We all respect each others time and energy spent on Foreman documentation.
+
+## Maintainers
+
+* help less experienced community members with git, Github, PRs, asciidoc, and asciidoctor.
+* expect a basic effort in contributions.
+* only merge PRs if the Github Action is green.
+* read the proposed change and make friendly and helpful comments.
+* ping community members with more experience if they are unsure about specific details.
+* try to review PRs in a timely manner.
+
+## Contributors
+
+* are able to make meaningful contributions to the Foreman documentation, regardless of their experience with git, asciidoc, writing documentation, or Foreman.
+* familiarise themselves with the [Pull Request Checklist](#Pull-Request-Checklist).
+* open additional issues if a contribution is incomplete.
+* are invited to review other PRs.
+
+## Pull Request Checklist
+
+* [ ] I am familiar with the conventions as specified in the [guides/README.md](guides/README.md) file.
+* [ ] Before pushing, I view my diff to master or the target branch and check for spelling mistakes, broken conflict resolutions, etc.
+* [ ] Before pinging others about my PR, I await the Github Actions to see if my branch builds.
+* [ ] If I add text that applies only to a specific downstream product, I notify others and give them a chance to request extending the `ifdef::[]` or `ifndef::[]` directive.
+* [ ] I don't add useless or trailing white space.
+* [ ] I put each sentence to its own line.
+* [ ] I write a meaningful commit message.
+See [seven rules for git commit messages](https://cbea.ms/git-commit/#seven-rules)
+* [ ] My change does not worsen the state of any build target.
```

Comment:
```suggestion
* [ ] I am aware that my commit message should address the open-source community and follow the 'Upstream First' approach.
* [ ] My change does not worsen the state of any build target.
```

Reply:
I have suggested a point that I learned recently.

Reply:
I tend to disagree, @mjivraja My thoughts:
what @maximiliankolb is suggesting here is something that is tangible. Ticking this would ensure build is broken by the changes and any contributor would commit to ensuring this as part of the agreement. 
You change is rather vague and doesn't have actionable consequences that are clear to the contributor. It also has terms that are only relevant when you're working in the downstream. Like, if you're a Foreman user who wants to contribute, there is no "Upstream" for these contributors. These are terms only used if there is a product that is built from an open source project.

Reply:
Thank you for your clarification @melcorr. More clear with my perspective on Foreman :+1: